service/RoleBandService.ts

```diff
@@ -0,0 +1,16 @@
+import { RoleBandResponse } from "../model/RoleBandResponse";
+const axios = require('axios');
+
+axios.defaults.baseURL = process.env.API_URL;
+
+module.exports.URL='http://localhost:8080/api/role-band-levels';
```

Comment:
If we want to keep this, let's make use of the baseURL property 

```suggestion
module.exports.URL = `${this.baseURL}/api/role-band-levels`;
```